src/main/java/NullabilityRefactoring.java

```diff
@@ -0,0 +1,372 @@
+import org.eclipse.jdt.core.dom.*;
+import org.eclipse.jdt.core.dom.rewrite.ASTRewrite;
+import java.util.*;
+
+class NullabilityRefactoring extends Refactoring {
```

Comment:
Is there a reason that this class isn't `public`? It's generally rare that you want a package-private class.